src/main/resources/the_warlordResources/localization/eng/WarlordMod-Power-Strings.json

```diff
@@ -5,5 +5,30 @@
       "Does something #b%1$s time.",
       "Does something #b%1$s times."
     ]
+  },
+
+  "the_warlord:BleedPower": {
+    "NAME": "Bleed",
+    "DESCRIPTIONS": [
+      "Some placeholder text?",
+      "More placeholder text!"
+    ]
+  },
+
+  "the_warlord:GashPower": {
+    "NAME": "Gash",
+    "DESCRIPTIONS": [
+      "Some placeholder text?",
+      "More placeholder text!"
+    ]
+  },
+
+  "the_warlord:NextTurnSyrettePower": {
+    "NAME": "dummy field",
```

Comment:
This one still needs a name. Maybe Prepare Syrette?

Reply:
Hi, Vertti here. I think it could be called Stimulation or Stimulus. That what I had in mind.